docs/chunk-deduplication.md

```diff
@@ -0,0 +1,131 @@
+**Remark：** This section of the picture and the associated DBSCAN algorithm description are referenced from : [https://en.wikipedia.org/wiki/DBSCAN](https://en.wikipedia.org/wiki/DBSCAN)
+
+#### Algorithm 2 Deduplication between different versions of the image (exponential smoothing algorithm)
+***
+**Basic principle:** Exponential smoothing algorithm is a method for time series data prediction and smoothing, the basic principle is to weighted average the data, give higher weight to the more recent repeated chunks, and constantly update the smoothing value, so the newer chunk has a greater impact on future forecasts, and the impact of older data will gradually weaken.
+
+**Input:** The training set and test set after deduplication in algorithm 1.
+
+**Output:** The chunk dictionary corresponding to each image.
+
+**Basic steps:**
+**1.** Divide all chunks in the training set into a new list according to the image_name, and each list corresponds to an image and all chunk sets in the image.
+
+**2.** The different versions inside each image are sorted chronologically, and each chunk is scored according to the Exponential Smoothing formula.
+$$S_0 =0 ,S_t = \alpha Y_{t-1} +(1- \alpha)S_{t-1} $$
+where, $\alpha=0.5$ , $Y_{t-1}$ indicates whether the chunk appeared in the previous image, 1 if it did, otherwise 0.
+
+**3.** Count the score for each chunk and select all chunks with a score greater than $THs$ as the chunk dictionary. Deduplicate the image version in the test set and calculate the storage space it occupies.
+
+**4.** Modify the value of $THs$ from 0.8 to 0.5 in steps of 0.05 and repeat steps 2 and 3 to generate multiple chunk dictionaries.
+
+**5.** Choose a chunk dictionary that minimizes the test set's storage space.
+***
+
+
+### Exponential smoothing algorithm test table
+Step 1: Download 10 OCI versions of an image and count the total size
+Step 2: Convert OCI to nydus image, and then count the total size after conversion
+Step 3: Select three versions of the image to generate chunkdict, use chunkdict to convert the remaining seven versions of the image, and then count the total size 
+dedulicating rate = (total_image_size(nydus) - total_image_size (nydus after dedulicating))/total_image_size(nydus)
+
+
+
+| image_name | version number | total_image_size(OCI) | total_image_size(nydus) | total_image_size (nydus after dedulicating) | chunkdict_image_size | dedulicating rate |
+|------------|----------------|-----------------------|-------------------------|---------------------------------------------|----------------------|-------------------|
+| redis      | 10             | 341.78                | 419.37                  | 319.48                                      | 41.87                | 23.82%            |
+| ubuntu     | 10             | 290.26                | 308.59                  | 140.28                                      | 30.8                 | 54.54%            |
```

Comment:
Let's split the table into two, one for "image info" and one for "deduplication result"